translations/base-de.yaml

```diff
@@ -7,17 +7,17 @@ steamPage:
 
         shapez.io ist ein entspanntes Spiel, in dem du Fabriken zur automatisierten Produktion von geometrischen Formen bauen musst.
 
-        Mit steigendem Level werden die Formen immer komplexer, und du musst dich auf der unendlich großen Karte ausbreiten. Das ist noch nicht alles, denn du musst exponentiell mehr produzieren,
+        Mit steigendem Level werden die Formen immer komplexer, und du musst dich auf der unendlich großen Karte ausbreiten. Das ist noch nicht alles, denn du musst exponentiell wachsen,
```

Comment:
Man muss in dem Spiel nicht `exponentiell wachsen` (Spielstil), sondern nur `exponentiell mehr abliefern/produzieren` (Ziel). Den Spielstil wollen nicht vorschreiben, deswegen nur das Ziel.